mono/metadata/icall-decl.h

```diff
@@ -195,6 +195,7 @@ ICALL_EXPORT void ves_icall_System_Runtime_RuntimeImports_ZeroMemory (guint8*, g
 #endif
 ICALL_EXPORT void ves_icall_System_Runtime_RuntimeImports_ecvt_s(char*, size_t, double, int, int*, int*);
 
```

Comment:
There is a lot more to be disabled. @lambdageek could someone look into that?

Reply:
@marek-safar Made a task to go through and clean up https://github.com/dotnet/runtime/issues/33533